virttest/utils_libvirt/libvirt_bios.py

```diff
@@ -0,0 +1,31 @@
+"""
+Libvirt BIOS related utilities.
+
+:copyright: 2022 Red Hat Inc.
+"""
+
+import logging
+
+
+LOG = logging.getLogger('avocado.' + __name__)
+
+
+def remove_items_for_efi(vmxml, params):
+    """
+    Remove efi firmware attribute and loader/nvram elements
+
+    :param vmxml: vm xml
+    :param params: dict for test
```

Comment:
params parameter is never used in the code implementation

Reply:
Updated, thanks.